fixtures/thread-id-test-worker.js

```diff
@@ -0,0 +1,3 @@
+'use strict'
+
+require('worker_threads').parentPort.postMessage(require('../thread-id'))
```

Comment:
I tried inlining this fixture into the tests using the `eval` option when constructing the Worker, but this `require` needs to be resolved from somewhere. Couldn't find a way to set that if it's possible.